sdks/java/core/src/main/java/org/apache/beam/sdk/io/AvroIO.java

```diff
@@ -283,17 +283,31 @@
         .build();
   }
 
+  /**
+   * Like {@link #read}, but reads each file in a {@link PCollection} of {@link
+   * FileIO.ReadableFile}, returned by {@link FileIO#readMatches}.
```

Comment:
",for example, returned by {@link FileIO#readMatches}" since manually constructing a PCollection of ReadableFile objects will work too.